src/feature/hs/hs_service.c

```diff
@@ -2289,19 +2289,12 @@ update_all_descriptors_intro_points(time_t now)
 /* Return true iff the given intro point has expired that is it has been used
  * for too long or we've reached our max seen INTRODUCE2 cell. */
 STATIC int
-intro_point_should_expire(const hs_service_intro_point_t *ip,
+intro_point_should_expire(hs_service_intro_point_t *ip,
```

Comment:
You could probably leave the const in here, if you want.

Reply:
Ouch that was left accidentally in from previous experimentations. Good catch.

Reply:
Fixed in 45f590055e67d122cc3e6cf88a88418fe80ddb0d.